dynamic-demo-plugin/src/components/ClusterOverview/Inventory.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { useTranslation } from 'react-i18next';
 import { Link } from 'react-router-dom';
-import { MonitoringIcon } from '@patternfly/react-icons/dist/esm/icons/monitoring-icon';
+import { MonitoringIcon } from '@patternfly/react-icons';
```

Comment:
We'll make sure we document the recommended way to do these imports.

Reply:
So I think it would be best to keep it tin the dynamic plugin README, even if its will make it bigger.